bounds_point:
  - x: -12.0
    y: -5.6
  - x: 3.16
    y: 10.0
  - x: 11.5
    y: 0.0
  - x: -3.84
    y: -12.72

min_z: -2.0
max_z: 2.0


team_color: "red"
map_color: "red"